x-pack/plugins/security_solution/public/hosts/components/first_last_seen_host/index.tsx

```diff
@@ -5,10 +5,9 @@
  */
```

Comment:
nit -> you are already in the folder of `hosts`, so we do know that `first_last_seen` is for a host. Do you agree?